<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgTestLevel</class>
 <widget class="QDialog" name="CDlgTestLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lgck-builder.qrc">
    <normaloff>:/images/lgckbuilder64ronnie2.png</normaloff>:/images/lgckbuilder64ronnie2.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Basic</string>
    </attribute>
    <widget class="QComboBox" name="cbSkill">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>2</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>94</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Lives</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>9</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Skill Level</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cContinue">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>127</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue until game over</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>38</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Score</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eScore">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>34</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>0000000</string>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>64</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Hit Points</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbLives">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>94</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="eHitPoints">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>64</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>000</string>
     </property>
     <property name="placeholderText">
      <string>50</string>
     </property>
    </widget>
    <zorder>cbSkill</zorder>
    <zorder>label_4</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>cContinue</zorder>
    <zorder>eScore</zorder>
    <zorder>label_3</zorder>
    <zorder>cbLives</zorder>
    <zorder>eHitPoints</zorder>
    <zorder>cbSkill</zorder>
    <zorder>label_4</zorder>
    <zorder>label</zorder>
    <zorder>cContinue</zorder>
    <zorder>label_2</zorder>
    <zorder>eScore</zorder>
    <zorder>label_3</zorder>
    <zorder>cbLives</zorder>
    <zorder>eHitPoints</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QCheckBox" name="cExternal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Run external process</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cInitGame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>93</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>InitGame</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbResolution">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="lgck-builder.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgTestLevel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgTestLevel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
